space-plugins/nextjs-starter/README.md

```diff
@@ -129,3 +129,23 @@ For more detailed information on Storyblok extensions, read the following guides
 
 - [Space Plugin](https://www.storyblok.com/docs/plugins/custom-application)
 - [OAuth 2.0 Authorization Flow](https://www.storyblok.com/docs/plugins/authentication-apps)
+
+
+## Troubleshooting
+
+If you have trouble setting up the development environment, please review the following:
+
+1. Ensure that the `.env.local` file is correctly set up with the following variables:
+  - `CLIENT_ID=`
+  - `CLIENT_SECRET=`
+  - `BASE_URL=`
+
+2. Ensure that the tunnel is correctly set up with the `BASE_URL`.
+
+3. Ensure that the extension settings inside Storyblok are correctly set up with the following properties:
+  - **Index to your page**: `{BASE_URL}`
+  - **Redirection endpoint**: `{BASE_URL}/api/connect/callback`
+
+4. Ensure that the extension settings inside Storyblok have the "Use App Bridge" option enabled.
```

Comment:
Since the App Bridge feature is currently available internally, I’m wondering if it would be the right time to make it accessible to everyone, or if we should keep it disabled by default in our starters so users don’t face potential restrictions or have a hard time troubleshooting possible issues. Wdyt @eunjae-lee?

Reply:
Oh sorry for not sharing earlier. It's publicly available now! :) 
https://github.com/storyblok/storyfront/pull/6354